build.gradle

```diff
@@ -86,6 +86,8 @@ subprojects {
     tasks.withType(Test) {
         systemProperty 'recreate', System.getProperty('recreate', 'false')
     }
+
+    tasks.check.dependsOn checkUnusedDependenciesMain, checkUnusedDependenciesTest
```

Comment:
If I use `checkUnusedDependencies` here, I get some inexplicable resolution errors. I don't want to go down the Gradle rabbit hole today so this is a workaround to at least enforce this check on the configurations we care about.

```
* What went wrong:
Could not determine the dependencies of task ':conjure-java-core:compileIntegrationInputJava'.
> Could not resolve all task dependencies for configuration ':conjure-java-core:integrationInputCompileClasspath'.
   > Could not find com.palantir.conjure.java.runtime:conjure-java-retrofit2-client:.
     Required by:
         project :conjure-java-core
```